api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -68,4 +68,10 @@ message QuicProtocolOptions {
   // If not specified the :ref:`default one configured by <envoy_v3_api_msg_extensions.quic.connection_id_generator.v3.DeterministicConnectionIdGeneratorConfig>` will be used.
   // [#extension-category: envoy.quic.connection_id_generator]
   core.v3.TypedExtensionConfig connection_id_generator_config = 8;
+
+  // Config how to get the server's preferred address to advertise and allow client migrating to it.
```

Comment:
nit: "Configure the server's preferred address to advertise so that client can migrate to it"